spec/ysh-funcs.test.sh

```diff
@@ -217,3 +217,80 @@ hit: 8
   ]
 ]
 ## END
+
+#### Varadic arguments, no other args
+func f(...args) {
+  = args
```

Comment:
Note that we might have to go back and change most of these to `json write (args)`, for stability

As part of the PyObject -> value_t rewrite, this format may change

Although it seems like people like it, so maybe we should try to keep it the same